app/services/zendesk/pvb_api.rb

```diff
@@ -0,0 +1,48 @@
+module Zendesk
+  class PVBApi
+    def initialize(zendesk_pvb_client)
+      self.zendesk_pvb_client = zendesk_pvb_client
+    end
+
+    def raise_ticket(ticket_attrs)
+      request do |client|
+        ZendeskAPI::Ticket.create!(client, ticket_attrs)
+      end
+    end
+
+    def cleanup_tickets
+      destroy_tickets(fetch_ticket_ids) until fetch_ticket_ids.empty?
+    end
```

Comment:
you want to:
1. store in a variable the result of the first call
2. check if empty
3. destroy tickets if any tickets left